<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Motion</class>
 <widget class="QWidget" name="Motion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motion</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>OutQuadMotion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_11">
       <property name="text">
        <string>OutBounceMotion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButton_22">
       <property name="text">
        <string>OutBackMotion</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>InQuadMotion</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="radioButton_14">
       <property name="text">
        <string>InBounceMotion</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="radioButton_25">
       <property name="text">
        <string>InBackMotion</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>InOutQuadMotion</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButton_12">
       <property name="text">
        <string>InOutBounceMotion</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="radioButton_23">
       <property name="text">
        <string>InOutBackMotion</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>OutCubicMotion</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="radioButton_16">
       <property name="text">
        <string>OutElasticMotion</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QRadioButton" name="radioButton_21">
       <property name="text">
        <string>OutCircMotion</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>InCubicMotion</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="radioButton_18">
       <property name="text">
        <string>InElasticMotion</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QRadioButton" name="radioButton_19">
       <property name="text">
        <string>InCircMotion</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>InOutCubicMotion</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="radioButton_13">
       <property name="text">
        <string>InOutElasticMotion</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QRadioButton" name="radioButton_24">
       <property name="text">
        <string>InOutCircMotion</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="radioButton_7">
       <property name="text">
        <string>OutQuartMotion</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="radioButton_15">
       <property name="text">
        <string>OutSineMotion</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QRadioButton" name="radioButton_20">
       <property name="text">
        <string>OutExpoMotion</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QRadioButton" name="radioButton_8">
       <property name="text">
        <string>InQuartMotion</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QRadioButton" name="radioButton_17">
       <property name="text">
        <string>InSineMotion</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QRadioButton" name="radioButton_26">
       <property name="text">
        <string>InExpoMotion</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QRadioButton" name="radioButton_9">
       <property name="text">
        <string>InOutQuartMotion</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QRadioButton" name="radioButton_10">
       <property name="text">
        <string>InOutSineMotion</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QRadioButton" name="radioButton_27">
       <property name="text">
        <string>InOutExpoMotion</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
